05-FITS/io_fits_ascii_overview.ipynb

```diff
@@ -331,27 +338,17 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "from astropy.visualization import imshow_norm, PercentileInterval, LogStretch\n",
     "from matplotlib import pyplot as plt\n",
-    "from matplotlib.colors import LogNorm\n",
     "\n",
     "%matplotlib inline"
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "plt.imshow(scidata, vmin=50, vmax=200)\n",
-    "plt.colorbar()"
-   ]
-  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For astronomical images, you often want the image scaling to be logarithmic. You can do this a number of ways, but the recommended was is to define the `norm` argument to be `matplotlib.colors.LogNorm()`."
+    "For astronomical images, you can use `astropy.visualization` to [normalize and stretch](https://docs.astropy.org/en/stable/visualization/normalization.html) the display. Here, we want it to be logarithmic."
```

Comment:
Interesting to introduce `astropy.visualization` here. It is shown in the WCS and Images section. I think there's no need to modify the notebook. I suggest to mention that more details will be shown in the later session.